README.md

```diff
@@ -622,6 +624,29 @@ $ tl add-text-embedding-feature input_file.csv \
 This command mainly wrap from kgtk's text-embedding functions.
 please refer to kgtk's readme page [here](https://github.com/usc-isi-i2/kgtk/blob/feature/embedding/kgtk/cli/text_embedding_README.md "here") for details.
 
+<a name="command_align-page-rank" />
+
+### [`align-page-rank`](#command_align-page-rank)` [OPTIONS]`
+Generates `aligned_pagerank` feature which is used in `vote-by-classifier` command.
+Aligned page rank means exact-match candidates retain page rank, fuzzy-match candidates receives 0.
+
+**Examples:**
+```bash
+tl align-page-rank candidates.csv > aligned_candidates.csv
+```
+
+**File Examples:**
+```
+|column|row|label          |kg_id     |pagerank          |method         |aligned_pagerank|
+|------|---|---------------|----------|------------------|---------------|----------------|
+|1     |0  |Citigroup      |Q219508   |3.988134e-09      |exact-match    |3.988134e-09    |
+|1     |1  |Bank of America|Q487907   |5.115590e-09      |exact-match    |5.115590e-09    |
+|1     |1  |Bank of America|Q50316068 |5.235995e-09 	 |exact-match    |5.235995e-09    |
+|1     |10 |BP             |Q100151423|5.115590e-09 	 |fuzzy-augmented|0.000000e+00    |
+|1     |10 |BP             |Q131755   |5.235995e-09 	 |fuzzy-augmented|0.000000e+00    |
+```
+
+
 <a name="command_check-extra-information" />
 
 ### [`check-extra-information`](#command_check-extra-information)` [OPTIONS]`
```

Comment:
@summer7xinting what is this command?

Reply:
this is the feature we use for classifier: aligned_pagerank which means exact-match-singletons retain their pagerank and fuzzy-match candidates get 0 because in previous experiments we see PR is not working so well and lowering our precision